queries/editorconfig/highlights.scm

```diff
@@ -0,0 +1,51 @@
+(comment) @comment @spell
+
+; consistency with toml and ini
+(section
+  (section_name) @type)
+
+(character_choice
+  (character) @constant)
+
+(character_range
+  start: (character) @constant
+  end: (character) @constant)
```

Comment:
```suggestion
(character) @character
```

Reply:
Here `character_choice` and `character_range` are the same as character classes and ranges, which in the queries of regex and gitignore are highlighted as `@constant`. Also, the `character` node in this grammar is more of a placeholder than something meaningful, I don't think is appropriate to highlight it like that.